corehq/motech/repeaters/models.py

```diff
@@ -939,6 +947,8 @@ class DataSourceUpdate(models.Model):
     class Meta:
         db_table = 'repeaters_datasourceupdate'
 
+    MAX_AGE = timedelta(days=93)
```

Comment:
so we are retaining 3 months of data?
Any reason for not using months=3 here?

Reply:
> 3 months of data?

Yeah, 3 months of records of changes. The source data is in the UCR of course, so if we ever need to get it into CommCare Analytics again, we can.

[timedelta](https://docs.python.org/3/library/datetime.html#timedelta-objects) doesn't have a `months` argument because a month is not a fixed amount of time.